pocs/tests/test_observatory.py

```diff
@@ -209,6 +257,13 @@ def test_standard_headers(observatory):
     assert headers['longitude'] == test_headers['longitude']
 
 
+def test_get_standard_headers_no_scheduler(observatory):
```

Comment:
We *do* want to be able to get headers without a scheduler, as mentioned in the `get_standard_headers` above.